src/patternfly/layouts/Page/styles.scss

```diff
@@ -1,110 +1,307 @@
+  --pf-l-page__main-section--BoxShadow:                     var(--pf-global--BoxShadow--lg-right);
+
+  // Main section horizontal nav
+  --pf-l-page__main-nav--PaddingTop:                        var(--pf-global--spacer--md);
+  --pf-l-page__main-nav--PaddingRight:                      0;
+  --pf-l-page__main-nav--PaddingBottom:                     var(--pf-global--spacer--md);
+  --pf-l-page__main-nav--PaddingLeft:                       var(--pf-global--spacer--xl);
+  --pf-l-page__main-nav--BackgroundColor:                   var(--pf-global--BackgroundColor--light-100);
+
+  // Main section modifiers
+  --pf-l-page__main-section--m-light--BackgroundColor:      var(--pf-global--BackgroundColor--light-100);
+  --pf-l-page__main-section--m-dark--BackgroundColor:       var(--pf-global--BackgroundColor--dark-transparent-200);
+
+  /* stylelint-enable */
+
+  // Base
   display: grid;
   min-height: 100vh;
-  padding: 0;
-  margin: 0;
-  grid-template-columns: var(--pf-l-page__sidebar--md--Width) 1fr;
-  grid-template-rows: auto 1fr;
+  grid-template-columns: 1fr;
+  grid-template-rows: max-content 1fr;
   grid-template-areas:
-    "header header"
-    "main main";
+    "header"
+    "main";
 
   @media (min-width: $pf-global--breakpoint--md) {
-    &.pf-m-expanded {
-      grid-template-areas:
-        "header header"
-        "sidebar main";
-    }
-  }
-
-  &__sidebar {
-    position: absolute;
-    top: 0;
-    bottom: 0;
-    left: 0;
-    width: var(--pf-l-page__sidebar--sm--Width);
-    grid-area: sidebar;
-    background-color: var(--pf-l-page__sidebar--BackgroundColor);
-    box-shadow: var(--pf-l-page__main-section--BoxShadow);
-    transform: translate3d(-100%, 0, 0);
-
-    .pf-m-expanded & {
-      transform: translate3d(0, 0, 0);
-    }
-
-    @media (min-width: $pf-global--breakpoint--md) {
-      position: static;
-      display: none;
-      width: 100%;
-      .pf-m-expanded & {
-        display: block;
-      }
-    }
+    grid-template-columns: max-content 1fr;
+    grid-template-areas:
+      "header header"
+      "nav main";
   }
 
+  // Header
   &__header {
-    // force the container follow the dark theme
+    // Force the container follow the dark theme
     @extend %pf-t-dark;
 
     grid-area: header;
     display: flex;
     flex-wrap: wrap;
+    align-items: flex-end;
+    max-width: 100%;
+    min-height: var(--pf-l-page__header--MinHeight);
+    padding-right: var(--pf-l-page__header--PaddingRight);
+    padding-left: var(--pf-l-page__header--PaddingLeft);
+    overflow-x: hidden;
 
     > * {
       display: flex;
       align-items: flex-end;
     }
 
+    // Toggle
+    &-sidebar-toggle {
+      padding: var(--pf-l-page__header-sidebar-toggle--Padding);
+      // Ensure button does not adjust positioning
+      margin-right: var(--pf-l-page__header-brand-link--MarginLeft);
+      margin-left: calc(var(--pf-l-page__header-sidebar-toggle--Padding) * -1);
```

Comment:
to conform to the var naming rule and so a customer can customize it, should this and the line above it have their own vars that apply to the property name they're modifying, and those vars could just be set to the values you're currently using? For example,  set `--pf-l-page__header-sidebar-toggle--MarginLeft: var(--pf-l-page__header-sidebar-toggle--Padding) * -1);` and use `margin-left: var(--pf-l-page__header-sidebar-toggle--MarginLeft);`?

Reply:
Removed.